src/App.jsx

```diff
@@ -1,104 +1,35 @@
+import { useEffect, useState } from 'react';
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
 
-export const App = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
+const getPosts = () => {
+  return postsFromServer.map(post => {
```

Comment:
Create object instead of function
```suggestion
const posts = postsFromServer.map((post) => ({
  ...post,
  user: ......,
  comments: .....,
}))
```